<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-PDM7OVM" ignored="0" name="RetryMultiple" tests="14" failures="4" timestamp="12 Apr 2023 09:26:17 GMT" time="0.199" errors="0">
  <testcase name="Test1" time="0.022" classname="runtimeRetryFailedTc.Test001">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.003" classname="runtimeRetryFailedTc.Test001">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.002" classname="runtimeRetryFailedTc.Test001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at runtimeRetryFailedTc.Test001.Test1(Test001.java:9)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase name="Test2" time="0.002" classname="runtimeRetryFailedTc.Test001">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.002" classname="runtimeRetryFailedTc.Test001">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.002" classname="runtimeRetryFailedTc.Test001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at runtimeRetryFailedTc.Test001.Test2(Test001.java:14)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
  <testcase name="Test1" time="0.008" classname="runtimeRetryFailedTc.Test002">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.002" classname="runtimeRetryFailedTc.Test002">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.003" classname="runtimeRetryFailedTc.Test002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at runtimeRetryFailedTc.Test002.Test1(Test002.java:11)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase name="Test2" time="0.002" classname="runtimeRetryFailedTc.Test002">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.002" classname="runtimeRetryFailedTc.Test002">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.004" classname="runtimeRetryFailedTc.Test002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at runtimeRetryFailedTc.Test002.Test2(Test002.java:16)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
  <testcase name="Test3" time="0.002" classname="runtimeRetryFailedTc.Test002">
    <skipped/>
  </testcase> <!-- Test3 -->
  <testcase name="Test3" time="0.005" classname="runtimeRetryFailedTc.Test002"/>
</testsuite> <!-- RetryMultiple -->
